sdk/aqueduct/tests/decorator_test.py

```diff
@@ -55,6 +55,14 @@ def test_handle_reserved_file_dependencies():
         )
 
 
+def test_missing_requirements_file():
+    with pytest.raises(FileNotFoundError):
```

Comment:
Can you add the "pytest.raises(..., match=...)" argument so we also confirm the error message here?

Reply:
done